app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsUserStats.kt

```diff
@@ -41,7 +41,7 @@ object SuggestedEditsUserStats {
                 }
     }
 
-    fun getPageViewsObservable(): Observable<Long> {
+    fun getPageViewsObservable(): Observable<Array<Any?>> {
```

Comment:
Maybe we can make this method to `private` and rename it to `getPageViewsObservableList`, and let it returns `ArrayList<Observable<MwQueryResponse>>`